cla_backend/apps/cla_butler/management/commands/delete_cases_without_personal_data.py

```diff
@@ -0,0 +1,16 @@
+# coding=utf-8
+from django.core.management.base import BaseCommand
+
+from cla_butler.tasks import DeleteOldData
+
+
+class DeleteCasesWithoutPersonalData(BaseCommand):
+
+    help = "Deleting cases that are over three years old and that dont have an excluded outcome_code"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--force", action="store_true", dest="force", help="Force running of housekeeping task")
+
+    def handle(self, *args, **options):
+        self.stdout.write("Deleting cases that are over three years old and that dont have an excluded outcome_code")
```

Comment:
```suggestion
        self.stdout.write("Deleting cases containing no personal data")
```

Reply:
Made this `self.help` as it's the same string as the help variable above.